protocols/user_data_processor.py

```
@@ -1,95 +1,104 @@
 # Real Protocol: User Data Processor
 import os
-import json
-import glob
 from datetime import datetime
 
+
 def task():
```

Ruff (0.11.9)

`task` is too complex (11 > 10)

(C901)